client/my-sites/plugins/main.jsx

```diff
@@ -383,12 +384,10 @@ export class PluginsMain extends Component {
 		return (
 			<Button
 				className="plugins__button"
-				compact
 				href={ browserUrl }
 				onClick={ this.handleAddPluginButtonClick }
 			>
-				<Gridicon icon="plus" />
-				<span className="plugins__button-text">{ translate( 'Add Plugin' ) }</span>
+				<span className="plugins__button-text">{ translate( 'Add plugin' ) }</span>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.